### DISCUSSION TITLE
CircuitPython | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1891] [Q&A]
# CircuitPython | Seeed Studio Wiki

Seeed Studio XIAO nRF52840 with CircuitPython

https://wiki.seeedstudio.com/XIAO-BLE_CircutPython/

**Comment by Riciard52** [comment]
now the available space to code is only 2MB instead of 32MB that was with the original FW any solutions ?
